.github/workflows-source/release-testing.yml

```diff
@@ -189,7 +189,7 @@ jobs:
         with:
           BAZEL_COMMAND: "test"
           BAZEL_TARGETS: "//rs/tests/dre:guest_os_qualification"
-          BAZEL_CI_CONFIG: "--config=systest --repository_cache=/cache/bazel"
+          BAZEL_CI_CONFIG: "--config=systest"
```

Comment:
I wonder if this shouldn't be `--config=ci` as well. But we should probably handle that in a different PR.